eve/main.yml

```
@@ -1553,3 +1556,269 @@ stages:
+          haltOnFailure: true
+      # Taint bootstrap node to allow deployment of example-solution-operator
+      - ShellCommand:
+          name: Taint Bootstrap node before deploying example-solution-operator
+          env:
+            KUBECONFIG: "/etc/kubernetes/admin.conf"
+            INFRA_TAINT: "node-role.kubernetes.io/infra:NoSchedule-"
+            BOOTSTRAP_TAINT: "node-role.kubernetes.io/bootstrap:NoSchedule-"
+          command: >
+            ssh -F ssh_config bootstrap -- "
+            sudo kubectl --kubeconfig ${KUBECONFIG} taint nodes --all ${BOOTSTRAP_TAINT} &&
+            sudo kubectl --kubeconfig ${KUBECONFIG} taint nodes --all ${INFRA_TAINT}
+            "
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Deploy example-solution
+          env:
+            SOLUTION_VERSION: "%(prop:example_solution_version)s"
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo bash
+            /var/tmp/metalk8s/solutions.sh add-solution --name
+            example-environment --solution example-solution --version
+            "${SOLUTION_VERSION}" --verbose
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Copy example-solution CRs & Bash script to Bootstrap node
+          env:
+            SSH_CONFIG: >-
+              eve/workers/openstack-multiple-nodes/terraform/ssh_config
+            HOST_CR_PATH: "examples/metalk8s-solution-example/example-operator/deploy/crds/examplesolution_v1alpha1_versionserver_cr.yaml"
+            GET_RESOURCE: "examples/metalk8s-solution-example/get_resource.sh"
+          command: >
+            scp -F $SSH_CONFIG "${HOST_CR_PATH}" bootstrap:/tmp &&
+            scp -F $SSH_CONFIG "${GET_RESOURCE}" bootstrap:/tmp
+          haltOnFailure: true
+      - ShellCommand:
+          name: Apply example-solution CRs on Bootstrap node
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo kubectl --kubeconfig /etc/kubernetes/admin.conf apply -f
+            /tmp/examplesolution_v1alpha1_versionserver_cr.yaml
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand: &check_k8s_resource_column_ssh
+          name: Verify example-solution-operator Deployment
+          env: &_env_check_k8s_resource_column_ssh
+            KUBECONFIG: "/etc/kubernetes/admin.conf"
+            NAMESPACE: "example-environment"
+            RESOURCE_TYPE: "deployments"
+            RESOURCE_NAME: "example-solution-operator"
+            RESOURCE_COLUMN: ":status.availableReplicas"
+            RESOURCE_VALUE: "1"
+            RETRIES: "10"
+            SLEEP: "3"
+          command: |
+            ssh -F ssh_config bootstrap
+            sudo bash /tmp/get_resource.sh
```

Should either be:
```suggestion
          command: |
            ssh -F ssh_config bootstrap \
            sudo bash /tmp/get_resource.sh
```
Or:
```suggestion
          command: >
            ssh -F ssh_config bootstrap
            sudo bash /tmp/get_resource.sh
```